cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -717,7 +717,32 @@ action.summary_index._name = cyences
 action.summary_index.category = Windows
 action.summary_index.alert_name = Windows - Windows Process Tampering Detected
 
-
+[Windows - Realtime Protection Failed/Disabled]
```

Comment:
Please update the existing savedsearch instead
`Windows Defender - Windows Defender RealTime Protection Disabled or Failed`

Reply:
We already have alert with above name. But it is not checking whether RealTime Protection is again enabled or not.

Reply:
Suggest any new name.